.travis.yml

```diff
@@ -88,4 +90,18 @@ jobs:
         - oc login $OPENSHIFT_URL --token=$OPENSHIFT_TOKEN
         - export CLOUDIGRADE_TOKEN=$(oc rsh -c cloudigrade-api $(oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=cloudigrade-api) scl enable rh-postgresql96 rh-python36 -- python manage.py drf_create_token $CLOUDIGRADE_USER | awk '{print $3}')
       script:
-        - xvfb-run py.test -vvv --driver Firefox integrade/tests/ui
+        - xvfb-run py.test -vvv integrade/tests/ui
+
+    - stage: test-cloudigrade
+      env:
+        - NAME=ui-edge
```

Comment:
Have you seen that pytest-selenium have the SauceLabs driver [1]? Also you can specify the capabilities using a file [2].

That said I was wondering if it would be better to come up with many different capabilities file and pick on or another depending on the browser and possible operating system.

[1] http://pytest-selenium.readthedocs.io/en/latest/user_guide.html#sauce-labs
[2] http://pytest-selenium.readthedocs.io/en/latest/user_guide.html#capabilities-files

Reply:
pytest-selenium should make easier to do "the separated configuration file for each browser" by using the Capabilities File (http://pytest-selenium.readthedocs.io/en/latest/user_guide.html#capabilities-files).

I am not sure if this will be possible but having the configuration handled by something other than integrade would allow us reuse things and that is why we started to use pytest-selenium. That said, do you think the capabilities file would help us avoiding having to handle browser configuration and have those config file hosted outside integrade python package?